src/thunderbots/software/ai/hl/stp/evaluation/player.h

```diff
@@ -0,0 +1,40 @@
+#ifndef AI_HL_STP_EVALUATION_PLAYER_H_
+#define AI_HL_STP_EVALUATION_PLAYER_H_
+
+#include "geom/point.h"
+
+/**
+ * Contain independent Evaluation function to evaluate whether player orientation at
+ * position is within threshold
+ */
+
+namespace AI
+{
+    namespace HL
+    {
+        namespace STP
+        {
+            namespace Evaluation
+            {
+                /**
+                 * Determines whether or not the player at position
+                 * is facing within threshold degrees of the specified target
+                 *
+                 * @param position current position of the robot
```

Comment:
Nit: The documentation should consistently use "robot" instead of switching between "robot" and "player".